WebServices/AzureNotificationHub/README.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Azure Notification Hub
+description: This sample demonstrates how to use Azure Notification Hubs to send push notifications to Xamarin Forms applications on Android and iOS.
+topic: sample
+languages:
+- csharp
+products:
+- xamarin
+technologies:
+- xamarin-forms
+urlFragment: webservices-azurenotificationhub
+---
+Azure Search
+============
+
+This sample demonstrates how to use Azure Notification Hubs to send push notifications to Xamarin Forms applications on Android and iOS.
+
+For more information about this sample see [Push notifications with Azure Notification Hub and Xamarin Forms](https://review.docs.microsoft.com/en-us/xamarin/xamarin-forms/data-cloud/push-notifications/azure-notification-hub).
```

Comment:
The final URL will be largely identical to this. Remove "review." and "en-us". So the URL becomes:

https://docs.microsoft.com/xamarin/xamarin-forms/data-cloud/push-notifications/azure-notification-hub

Reply:
Also, really the readme should point out that users need to replace google-services.json with their own file.